tools/istio-iptables/pkg/config/config.go

```diff
@@ -77,5 +79,6 @@ func (c *Config) Print() {
 	fmt.Printf("OUTBOUND_PORTS_EXCLUDE=%s\n", c.OutboundPortsExclude)
 	fmt.Printf("KUBEVIRT_INTERFACES=%s\n", c.KubevirtInterfaces)
 	fmt.Printf("ENABLE_INBOUND_IPV6=%t\n", c.EnableInboundIPv6)
+	fmt.Printf("DNS_SERVERS=%s,%s\n", c.DNSServersV4, c.DNSServersV6)
```

Comment:
auto cherry pick tried to add
`	fmt.Printf("DNS_CAPTURE=%t\n", c.RedirectDNS)`
which was done as a different feature